pkg/resource/logging-secret/alloy/logging.alloy.template

```diff
@@ -0,0 +1,265 @@
+discovery.kubernetes "kubernetes_pods" {
```

Comment:
I'm pretty sure we will have to change this but this is a translation of the current promtail config

Reply:
Yes this config file is a translation of the current promtail config.